projects/ngx-launcher/src/lib/launcher.component.ts

```diff
@@ -41,6 +41,12 @@ export class LauncherComponent implements AfterViewInit, OnInit {
    * Setting the flag to show dependency editor as internal feature
    */
   @Input() depEditorFlag = false;
+
+  /**
+   * Setting the flag to show or hide the View pipeline and Open IDE button
+   */
+  @Input() nextButtons = false;
```

Comment:
do we really need another flag to enable or disable this?

Reply:
Yes, I think we need this flag to hide and show the buttons.